Makefile

```diff
@@ -164,7 +164,7 @@ system-test:
 
 system-test-deps: | system_test/aest_hard_fork_SUITE_data/db/mnesia_ae-uat-epoch_backup.tar
 	$(info The downloaded DB backup is not checked for freshness.)
-	docker pull aeternity/epoch:v0.11.1
+#	docker pull aeternity/epoch:latest
```

Comment:
`epoch:local` cannot be pulled, this is commented out because right now the test doesn't sync with latest because the genesis block changed. When this is not the case anymore this should be uncommented and latest should be used in `new_node_joins_network` again.